pom.xml

```
@@ -206,6 +208,27 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>clirr-maven-plugin</artifactId>
+        <version>2.8</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
```

I think an "old" version for comparison needs to be defined here (the `feel.version.old` property needs to be defined inside the properties tag):
```suggestion
        <configuration>
          <comparisonVersion>${feel.version.old}</comparisonVersion>
```